package.json

```diff
@@ -10,8 +10,10 @@
   "devDependencies": {
     "eslint": "^5.11.1",
     "eslint-config-bliss": "^3.1.2",
+    "husky": ">=1",
```

Comment:
Not concerned with breaking changes to the configuration?

Reply:
Good catch. This was added by default by [lint-staged's setup script](https://github.com/okonet/lint-staged#installation-and-setup). Changing this to minor server.